src/entities/twinFlowTransition/api/types.ts

```diff
@@ -21,17 +21,14 @@ export type TwinFlowTransitionCreateRq =
 export type TwinFlowTransitionUpdateRq =
   components["schemas"]["TransitionUpdateRqV1"];
 
-export type TwinFlowTransitionApiFilterFields =
+export type TF_TransitionFilterKeys =
```

Comment:
I personally don't like abbreviations like this, they are more difficult to read and refactor, especially for new developers on the project.